docs/concepts/aggregate-stand.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Aggregate Stand
```

Comment:
Let's put it off for now. It's just `Stand` now as it handles more than just aggregates.